apps/ledger-live-desktop/src/renderer/components/OnboardingAppInstall/index.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { Flex, Text, Button } from "@ledgerhq/react-ui";
+import { Device } from "@ledgerhq/live-common/hw/actions/types";
+import { DeviceModelInfo } from "@ledgerhq/types-live";
+import { useTranslation } from "react-i18next";
+import { getDeviceModel } from "@ledgerhq/devices";
+import useFeature from "@ledgerhq/live-common/featureFlags/useFeature";
+
+import DefaultAppsIllustration from "./DefaultAppsIllustration";
+import RestoreAppsIllustration from "./RestoreAppsIllustration";
+import CancelModal from "./CancelModal";
+import InstallSetOfApps from "./InstallSetOfApps";
+
+const fallbackDefaultAppsToInstall = ["Bitcoin", "Ethereum", "Polygon"];
+
+type Props = {
+  device?: Device | null | undefined;
```

Comment:
This probably should not be an optional property. There is no use case where you would use this component without a `device` prop. It would make sense if there was some internal logic to retrieve a `device` object somehow, but it's not the case here.
Having it be null or undefined is enough.

Reply:
Done